docs/discord/discord-readme.md

```diff
@@ -64,10 +64,6 @@ TEXT CHANNELS: These consist of the server's main text channels, that are for di
 
 - **#public-speaking**: improve your soft skills, public speaking, and build communities
 
-TECH: Channels regarding specific areas
```

Comment:
Removed tech channels attempt looks legit to me, would approve, unless you'd like to make further changes and expand the scope of the PR itself?

Reply:
I like to keep PRs small if possible, so happy to merge and further changes can come in another PR